code-challenges.js

```diff
@@ -9,67 +9,248 @@
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45] [10, 25, -40, 100, 160]
 
+
+//setup function that takes an array as the argument 
+//using a for loop multiply each number by 5
+
+
 var myForNumbers1 = [3, 7, 0, 6, -9]
 var myForNumbers2 = [2, 5, -8, 20, 32]
 
+const multiplyByFive = (array) => {
+let multiplied = [];
+for (let i =0  ; i < array.length ; i++){
+    multiplied.push (array[i] * 5);
+}
+return multiplied;
+}
 
+console.log(multiplyByFive(myForNumbers1))
+console.log(multiplyByFive(myForNumbers2))
 
 
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [435, 30, 140, 80, 10] [40, -35, 0, 30, 60]
 
-var myMapNumbers1 = [87, 6, 28, 16, 2]
-var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+//set up function that takes an array as the argument 
+//returns new a array with values mulitplied by 5
+//using .map()
 
 
+var myMapNumbers1 = [87, 6, 28, 16, 2]
+var myMapNumbers2 = [8, -7, 0, 6, 12]
 
+const mulitpiedArrayBy5 = (array) => {
+    return array.map (value =>{
+        return value * 5
+    })
+}
+console.log(mulitpiedArrayBy5(myMapNumbers1))
+console.log(mulitpiedArrayBy5(myMapNumbers2))
 
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
+//set up function that takes a string as the argument
+//has to use filter to create new subset array
+// use the .split to split up the string into array
+//use join to set up new array without vowela
+
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
 
+const vowelEssString = (array) =>{
+    let splitString = array.split("")
+    return splitString.filter (value =>{
+    return value !== "a" && value !=="e" && value !== "i" && value !== "o" && value !== "u" && value !== "I"
+    }).join('')
+}
+
+console.log(vowelEssString(stringWithVowels1))
+
+console.log(vowelEssString(stringWithVowels2))
 
 
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string" "mCdngMstr"
 
+//setup function that will take a var 
+//retrun if it is a string 
+//if it is a string remove the vowels
+
+
+//????? I tried everything I could think of but I could not get the same affect i got in my last exercise. 
+//????? It will inout the string but it does not remove the vowels. 
+
+
 var refactorTester1 = true
 var refactorTester2 = 42
 var refactorTester3 = "IAmACodingMaster"
 
+const vowelRemover = (i) =>{
+    //let newVar = [i]
+    let splitString = []
+    if (typeof i  !== "string")
+    return `${i} is not a string`
+    else if(typeof i  === "string")
```

Comment:
looks like the else if here isn't quite needed because it is the exact opposite of `typeof i !== "string"`
```suggestion
    else {
    ...
    }
```